### ISSUE TITLE
[Bug]: failed to load wineusb.sys.so

### ISSUE THREAD

**Issue by NOOBNUBY** [Whisky-App/Whisky#575]
### Description

I get this error when downloading Valorant and the Riot client won't open.

### Steps to reproduce

1. Start whisky
2. make new bottle window 10
3. start install valorant.exe

### Expected behaviour

starting riot client UI

### Logs

```bash
0068:err:module:open_builtin_so_file failed to load .so lib "/Users/noobnuby/Library/Application Support/com.isaacmarovitz.Whisky/Libraries/Wine/lib/wine/x86_64-unix/wineusb.sys.so"
0068:err:ntoskrnl:ZwLoadDriver failed to create driver L"\\Registry\\Machine\\System\\CurrentControlSet\\Services\\wineusb": c0000142
```

### What version of Whisky are you using?

2.1.1

### What version of macOS are you using?

Sonoma (macOS 14)

### Issue Language

- [X] Yes my issue is written in English

**Comment by mavethee** [comment]
Valorant won't load as Riot is using garbage kernel level malware as anti-cheat and does not provide support for Wine.

If you want to enjoy this malware on your computer, get Windows machine sadly.

Any kinds of hypervisors to run under virtual machine won't work either as this malware checks if it runs under VM.

TL;DR _Stop supporting garbage companies_
